bank_statement_journal_items/models/account_bank_statement.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+from odoo.tools.translate import _
```

Comment:
```suggestion
from odoo import _, models
```
Nitpicking, but you can import `_` directly from `odoo`